_docs/ops/runbook/troubleshooting-logsearch.md

```diff
@@ -84,21 +158,16 @@ input {
   s3 {
     bucket => ":bucket:"
     region => ":region:"
-    access_key_id => ":access_key_id:"
-    secret_access_key => ":secret_access_key:"
 
     type => "syslog"
     sincedb_path => "/var/vcap/data/ingestor_syslog/tmp/s3_import.sincedb"
   }
 }
 ```
 
-The values for `:bucket:` and `:region:` can be found in [cg-provision](https://github.com/18F/cg-provision/blob/main/terraform/modules/cloudfoundry/buckets.tf#L25-L30)
-or retrieved from the bosh manifest:
-
-```sh
-bosh -d logsearch manifest
-```
+The values for `:bucket:` and `:region:` can be retrieved from the bosh manifest
```

Comment:
Did we want to maybe move this earlier in the sequence so the operator doesn't have to either 1) jump into and out of their `bosh ssh` session on the ingestor or 2) create a second jumpbox session to grab the values?